crates/net/network/src/session/active.rs

```diff
@@ -329,8 +334,13 @@ impl ActiveSession {
     /// Returns the message if the bounded channel is currently unable to handle this message.
     #[allow(clippy::result_large_err)]
     fn try_emit_request(&self, message: PeerMessage) -> Result<(), ActiveSessionMessage> {
-        match self
-            .to_session_manager
+        let sender = self.to_session_manager.inner().get_ref();
+        if sender.is_none() {
+            return Ok(())
+        }
+
+        match sender
+            .unwrap()
```

Comment:
style:

let's use `let Some(sender) ... {else return Ok(())}`
pattern here

Reply:
I.. have not use let - else before. This is wonderful.